tests/taiga/taiga_test.go

```diff
@@ -1,15 +1,14 @@
-//go:build integration
-// +build integration
-
 package test
 
 import (
+	"log"
```

Comment:
same comments :

- I think it should be moved into testing.M function instead and have it ready before everything
- rename `sk`
- avoid using `log`. we can use `t.LogX` instead
- res should be `output`, not to be confused with the status code